sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -574,4 +578,49 @@ internal class RadarApiClient(
         })
     }
 
+
+    internal fun getContext(
+        location: Location,
+        callback: RadarContextApiCallback
+    ) {
+        val publishableKey = RadarSettings.getPublishableKey(context)
+        if (publishableKey == null) {
+            callback.onComplete(RadarStatus.ERROR_PUBLISHABLE_KEY)
+
+            return
+        }
+
+        val queryParams = StringBuilder()
+        queryParams.append("coordinates=${location.latitude},${location.longitude}")
```

Comment:
had a similar comment on the iOS PR - we could potentially move `GET` param construction in the url into the `apiHelper` layer, not urgent at all